### ISSUE TITLE
[SCREEN READER, BEST PRACTICE]: Storybook - More accurate and descriptive page titles

### ISSUE THREAD

**Issue by jamigibbs** [department-of-veterans-affairs/vets-design-system-documentation#1456]
## Description
The design system Storybook instance does not seem to have accurate or descriptive page title on the initial landing page.
 
When the Storybook page first loads, the title is set to "Webpack App" and screenreaders reads this as:

<img width="823" alt="Screenshot 2023-01-20 at 2 47 56 PM" src="https://user-images.githubusercontent.com/872479/213803154-768e483e-8b70-4161-a915-eaad23a90fb3.png">

I believe it's because this title value is being set on the page:

<img width="866" alt="Screenshot 2023-01-20 at 3 06 09 PM" src="https://user-images.githubusercontent.com/872479/213805348-bd315719-51a7-4839-b915-d39ee6ad14a3.png">

This might be easily resolved by updating the following Storybook configuration: https://github.com/jantimon/html-webpack-plugin#options See also: https://github.com/storybookjs/storybook/issues/17709

The other storybook pages may also need to be evaluated for accuracy and usefulness. Review this reference for what kind of text should be added to page `<title>'

- https://accessibility.iu.edu/creating-content/web-content/titles.html
- https://dequeuniversity.com/checklists/web/page-title
- https://www.w3.org/WAI/WCAG22/Understanding/page-titled.html

## Tasks
- [ ] Update the text "Webpack App" text to be more helpful.
- [ ] Check internal pages as well to make sure the titles make sense there too.

## Acceptance Criteria
- [ ] The initial page title says something more descriptive than "webpack app".
- [ ] Internal page titles have been checked and are all relevant/accurate.
